CHANGELOG.asciidoc

```diff
@@ -17,10 +17,10 @@
 ////
 
 [float]
-[[lambda-unreleased]]
-=== Unreleased
+[[lambda-1.4.0]]
+=== 1.4.0 - 2023/05/03
```

Comment:
Should we leave unreleased and add 1.4.0?

Reply:
That would add the unreleased section in the tag, I was planning to add it after releasing 1.4.0

Reply:
:+1: got it